tardis/io/decay.py

```diff
@@ -86,4 +86,37 @@ def decay(self, t):
         
         df = IsotopeAbundances.from_materials(decayed_materials)
         df.sort_index(inplace=True)
-        return df 
\ No newline at end of file
+        return df 
+
+    def merge_isotopes(self):
```

Comment:
I'm not happy with the name of this function, maybe we should call it `as_atoms`? That's short and reflects that the output is only atoms instead of isotopes.